staging/src/k8s.io/api/auditregistration/v1alpha1/types.go

```diff
@@ -96,14 +96,41 @@ type AuditSinkList struct {
 
 // Policy defines the configuration of how audit events are logged
 type Policy struct {
-	// The Level that all requests are recorded at.
+	// Level is the base Level that all requests are recorded at.
 	// available options: None, Metadata, Request, RequestResponse
 	// required
 	Level Level `json:"level" protobuf:"bytes,1,opt,name=level"`
 
 	// Stages is a list of stages for which events are created.
+	// If no stages are given nothing will be logged
 	// +optional
 	Stages []Stage `json:"stages" protobuf:"bytes,2,opt,name=stages"`
+
+	// PolicyRules define how classes should be handled.
+	// A request may fall under multiple audit classes.
+	// Rules are evaluated in order (first matching wins).
+	// Rules override the top Level & Stage.
+	// Unmatched requests use the top level rule.
+	// +optional
+	Rules []PolicyRule `json:"rules" protobuf:"bytes,3,opt,name=rules"`
+}
+
+// PolicyRule defines how a class is handled per sink
+type PolicyRule struct {
+	// AuditClassName of the AuditClass object. This rule matches requests that are
+	// classified with this AuditClass
+	AuditClassName string `json:"auditClassName" protobuf:"bytes,1,opt,name=auditClassName"`
```

Comment:
I feel like it might be slightly clearer if we can sneak the word "match" in the name somehow. "MatchingAuditClass"? "ForAuditClass"? "WithAuditClass"?